### ISSUE TITLE
Feature request: Open link in the background tab

### ISSUE THREAD

**Issue by cldotdev** [iorate/uSuperDrag#1]
It would be nice to add an action of opening link in the background tab.

1.  Drag to left: Open in background tab
2. Drag to right: Open in front tab

**Comment by imtrobin** [comment]
yes, I would like to see that option to open the tab in background instead of switching over immediately

**Comment by iorate** [comment]
Implemented in [02f38f8](https://github.com/iorate/uSuperDrag/commit/02f38f848f7e0376f9eeed4e7404d1f9a991a968): an "Open new tab in background" option, plus Ctrl (Command on Mac) while dropping to invert it per-drop. I have no intention of adding anything beyond this — drag-direction-based behavior or other variations will not be implemented. Closing.
